test/unit/job_invocation_report_template_test.rb

```diff
@@ -0,0 +1,66 @@
+require 'test_plugin_helper'
+
+class JobReportTemplateTest < ActiveSupport::TestCase
+  class FakeTask < OpenStruct
+    class Jail < Safemode::Jail
+      allow :action_output, :ended_at
+    end
+  end
+
+  context 'job invocation report template' do
+
+    describe 'correct template is available' do
+      it 'settings select includes only report templates with name job_id' do
+        FactoryBot.create(:report_template, name: 'Template 1')
+        correct_template = FactoryBot.create(:report_template, name: 'Job Invocation Report Template')
+        FactoryBot.create(:template_input, template: correct_template, input_type: 'user', name: 'job_id')
+        templates = Setting::RemoteExecution.job_invocation_report_templates_select
+
+        assert_include templates, 'Job Invocation Report Template'
+      end
+    end
+
+    describe 'test correct result for report template' do
+      let(:fake_outputs) do
+        [
+          { output_type: 'stderr', output: "error" },
+          { output_type: 'stdout', output: "output" },
+          { output_type: 'stdebug', output: "debug" },
+        ]
+      end
+      let(:fake_task) { FakeTask.new(action_output: fake_outputs, ended_at: Time.new(2020, 12, 1, 0, 0, 0)) }
+
+      it 'should process the result of task to template' do
+        job_invocation = FactoryBot.create(:job_invocation, :with_task)
+        JobInvocation.any_instance.expects(:sub_task_for_host).returns(fake_task)
+
+        template_source = File.read(File.expand_path(Rails.root + "app/views/report_templates/jobs_invocations.erb"))
```

Comment:
Wrong path to the template.
`app/views/unattended/report_templates/jobs_-_invocation_report_template.erb`